content/resources/help-centre/software-tips/package-manager.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Package manager
+---
+
+## What is a package manager?
+
+Much of the software that is mentioned on this website includes package manager
+installation instructions, if they are available. But what is a package manager?
+Essentially, it is a piece of software that lets us easily and conveniently
+download and install applications (known as packages), and keep those packages
+up to date. A package manager runs in the [command line]({{% ref
+"command-line.md" %}}), so check out that help page first if you are unfamiliar
+with using the command line. 
+
+You might be asking, "why would I want to use a package manager?". The biggest
+advantage of a package manager is the convenience. For example, installing
+FFmpeg manually can be a difficult process. Especially when you have to repeat
+the installation process on every computer you want to use. But with a package
+manager, a single command can get you up and running in no time at all:
+
+```ps 
+> choco install ffmpeg
+```
+
+That means less time fussing around with installations, and more time for you to
+focus on the things that matter. Besides FFmpeg, there are many useful software
+packages that are commonly used for ecoacoustics which can be installed with
+package managers.
+
+The other great thing about package managers is that they take care of your
+[PATH variable]({{% ref "path.md" %}}), ensuring that anything you
+install will be available and ready to use directly from the [command line]({{%
+ref "command-line.md" %}}).
+
+## Get started
+
+The type of package manager you use will depend on your operating system. We
+recommend three main options: 
+
+- [Chocolatey](https://chocolatey.org/) on Windows
+- [Homebrew](https://brew.sh/) on macOS 
+- [Snap](https://snapcraft.io/) on Linux
+
+### How to install
```

Comment:
seems like this choco guide (and a brew guide) should be a sub-pages

i see the tabs... but we're not really showing alternate functionality...